proc/merge/merge.go

```diff
@@ -173,43 +170,42 @@ func (m *Proc) Pop() interface{} {
 }
 
 type puller struct {
```

Comment:
As in `proc/combine`, I think we can simplify all of this if we change things around so this type is just a `zbuf.Puller` that wraps its upstream/parent in a goroutine, but let's worry about that later.

Reply:
issue #3453